generator/transit_generator.cpp

```diff
@@ -43,20 +59,82 @@ string GetFileName(string const & filePath)
   return name;
 }
 
-/// \brief Reads from |root| (json) and serializes an array to |serializer|.
-/// \param handler is function which fixes up vector of |Item|(s) after deserialization from json
-/// but before serialization to mwm.
 template <class Item>
-void SerializeObject(my::Json const & root, string const & key, Serializer<FileWriter> & serializer,
-                     function<void(vector<Item> &)> handler = nullptr)
+void DeserializeFromJson(my::Json const & root, string const & key, vector<Item> & items)
 {
-  vector<Item> items;
-
+  items.clear();
   DeserializerFromJson deserializer(root.get());
   deserializer(items, key.c_str());
+}
+
+void DeserializerGatesFromJson(my::Json const & root, string const & mwmDir, string const & countryId,
```

Comment:
may be 'Deserialize' verb instread noun 'Deserializer'?

Reply:
+1

Reply:
Sure. Thanx.